browser/constants.js

```diff
@@ -419,6 +419,15 @@ const KOMMUNER =[
     }
 ]
 
+
+const Variants = {
```

Comment:
Move Enum `Variants` to `/components/shared/contants/variants.js`
We will build and use some extra Enum constants to control the rendering og our components